neqo-http3/src/connection_client.rs

```diff
@@ -1852,7 +1852,7 @@ mod tests {
             if let ConnectionEvent::RecvStreamReadable { stream_id } = e {
                 if stream_id == request_stream_id {
                     // Read the DATA frame.
-                    let mut buf = [1_u8; 0xffff];
+                    let mut buf = vec![1_u8; Connection::stream_recv_buffer_size()];
```

Comment:
Why is this an associated function and not an exported constant?

Reply:
I think exported constant is better. I'll follow up with another PR.